Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -970,6 +975,12 @@ private void UpdatePointer(PointerData pointerData)
                     hitResult3d.Clear();
                     QueryScene(pointerData.Pointer, raycastProvider, prioritizedLayerMasks, hitResult3d, maxQuerySceneResults, focusIndividualCompoundCollider);
 
+                    if ((gazeProviderPointingData?.Pointer != null) &&
+                        (pointerData.Pointer.PointerId == gazeProviderPointingData.Pointer.PointerId))
+                    {
+                        gazeHitResult = hitResult3d;
```

Comment:
Can you also add a comment on why we're setting gazeHitResult here and the overwriting it down below (where it's already set today?)

Reply:
good question. it has been a while since i worked on this code... i will confirm why/if this needs to be there shortly.

Reply:
Is this just an artifact of what looks like a stabilzation -> mrtk_dev merge, or is this new to this change?

Reply:
i will double check. it may be an artifact

Reply:
this may take a bit to unwind. i will update soon

Reply:
this appears to have been a merge artifact... testing shortly

Reply:
i removed the code highlighted and the app runs as expected.... apparent merge artifact.